src/components/ArticlesList.jsx

```diff
@@ -9,22 +10,29 @@ export default function ArticlesList() {
   useEffect(() => {
     getArticles().then((res) => {
       setArticles(res);
-      setIsLoading(false)
+      setIsLoading(false);
     });
   }, [articles]);
 
-
-  if(isLoading){
-  return (<h2>Loading...</h2>)
-  } else{
-   return (
-    <section>
-      <ol>
-        {articles.map((article) => {
-          return <Article article={article} key={article.article_id} />;
-        })}
-      </ol>
-    </section>
-  ); 
+  if (isLoading) {
+    return <h2>Loading...</h2>;
+  } else {
+    return (
+      <section>
+        <ol>
+          {articles.map((article) => {
+            return (
+              <section key={article.article_id}>
```

Comment:
maybe you don't need `section`  at all here, and can instead put the key on the `li` element instead